### PR TITLE
OG tags, previews for social

### PR DESCRIPTION
**Pull request by march08**
Screen Shot 2021-04-20 at 13 52 10
Screen Shot 2021-04-20 at 13 52 02

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,12 +1,64 @@
...
+    />
+    <meta name="twitter:site" content="@flexpool_io" />
+    <meta
+      name="twitter:image:src"
+      content="https://beta.flexpool.io/og/flexpool_og.jpg"
+    />
+    <!-- Open Graph general (Facebook, Pinterest & Google+) -->
+    <meta
+      property="og:title"
+      content="The most advanced mining pool |Flexpool.io"
```
Comment (AlexSSD7) [inline public/index.html:46]: Should be capitalized "The Most Advanced Mining Pool" + missing space after "|"

File: public/index.html [hunk 2]
```diff
@@ -1,12 +1,64 @@
...
     />
+    <meta
+      itemprop="image"
+      content="https://beta.flexpool.io/og/flexpool_og.jpg"
+    />
+    <!-- Twitter -->
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta
+      name="twitter:title"
+      content="The most advanced mining pool | Flexpool.io"
```
Comment (AlexSSD7) [inline public/index.html:32]: Should be capitalized "The Most Advanced Mining Pool"

File: public/index.html [hunk 3]
```diff
@@ -1,12 +1,64 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <!-- OG, TW -->
```
Comment (AlexSSD7) [inline public/index.html:4]: What's the purpose of this?

File: public/index.html [hunk 4]
```diff
@@ -1,12 +1,64 @@
...
 
+    <!-- COMMON TAGS -->
+    <meta charset="utf-8" />
+    <title>The most advanced mining pool | Flexpool.io</title>
```
Comment (AlexSSD7) [inline public/index.html:8]: Should be capitalized "The Most Advanced Mining Pool"

File: public/index.html [hunk 5]
```diff
@@ -1,12 +1,64 @@
...
     <meta
       name="description"
-      content="The mining pool that offers yeou the best mining experience. We build the future of mining pools here."
+      content="The mining pool that offers you the best mining experience. We build the future of mining pools here."
+    />
+    <meta name="image" content="https://beta.flexpool.io/og/flexpool_og.jpg" />
+    <!-- Schema.org for Google -->
+    <meta
+      itemprop="name"
+      content="The most advanced mining pool | Flexpool.io"
```
Comment (AlexSSD7) [inline public/index.html:18]: Should be capitalized "The Most Advanced Mining Pool"

File: public/index.html [hunk 6]
```diff
@@ -1,12 +1,64 @@
...
+      content="https://beta.flexpool.io/og/flexpool_og.jpg"
+    />
+    <meta property="og:url" content="https://beta.flexpool.io" />
+    <meta property="og:site_name" content="Flexpool.io" />
+    <meta property="og:type" content="website" />
+
+    <!-- END OG, TW -->
     <meta
       name="keywords"
       content="ethereum mining pool,flexpool,best ethereum mining pool,fastest ethereum mining pool,ethereum pool"
```
Comment (AlexSSD7) [inline public/index.html:64]: `cryptocurrency,mining pool,flexpool` - this please.

**Review by AlexSSD7 — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 3 above)
Comment (march08) [reply public/index.html:4]: What do you mean? It's just a html comment just to add clarity for sections

Comment (AlexSSD7) [reply public/index.html:4]: Yes, but what is `TW` :sweat_smile:? Doesn't seem to be clear.

**Review by AlexSSD7 — APPROVED** [review]
LGTM

**Pull request merged**